hrpsys_ros_bridge/launch/hrpsys_ros_bridge.launch

```diff
@@ -102,6 +102,7 @@
     <rtconnect from="DataLoggerServiceROSBridge.rtc:DataLoggerService" to="log.rtc:DataLoggerService" subscription_type="new"/>
     <rtconnect from="SequencePlayerServiceROSBridge.rtc:SequencePlayerService" to="seq.rtc:SequencePlayerService" subscription_type="new"/>
     <rtconnect from="ForwardKinematicsServiceROSBridge.rtc:ForwardKinematicsService" to="fk.rtc:ForwardKinematicsService" subscription_type="new"/>
+    <rtconnect from="es.rtc:emergencyMode" to="HrpsysSeqStateROSBridge0.rtc:emergencyMode" subscription_type="new"/>
```

Comment:
Could you move this line into the region inside `<group if="$(arg USE_EMERGENCYSTOPPER)" >`?
Currently, this code does not work in Stable rtc environments because `rtconnect` cannot find `es.rtc`.
In that case, we use flags like `USE_EMERGENCYSTOPPER`.